### ISSUE TITLE
Own implementation of RequestRunner

### ISSUE THREAD

**Issue by deadly-cat** [stephanenicolas/robospice#406]
Hi, I need to replace DefaultRequestRunner to own. But as I see SpiceService.createRequestRunner() now is private method.
Is it possible to make it protected?

**Comment by softwaremaverick** [comment]
That would be easy enough to do and I doubt anyone would complain. Although you should probably make it protected as opposed to public since you'll only use it in your own spice service.

On 17 February 2015 19:50:26 GMT+00:00, Igor Zubenko <notifications@github.com> wrote:
>Hi, I need to replace DefaultRequestRunner to own. But as I see
>SpiceService.createRequestRunner() now is private method.
>Is it possible to make it protected?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/stephanenicolas/robospice/issues/406

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by softwaremaverick** [comment]
Just realised you said protected already so never mind :-)

Just submit a commit with your changes and someone will be right with you!

On 17 February 2015 20:12:48 GMT+00:00, Andrew Clark <andrew@dync.co.uk> wrote:
>That would be easy enough to do and I doubt anyone would complain.
>Although you should probably make it protected as opposed to public
>since you'll only use it in your own spice service.
>
>On 17 February 2015 19:50:26 GMT+00:00, Igor Zubenko
><notifications@github.com> wrote:
>>Hi, I need to replace DefaultRequestRunner to own. But as I see
>>SpiceService.createRequestRunner() now is private method.
>>Is it possible to make it protected?
>>
>>---
>>Reply to this email directly or view it on GitHub:
>>https://github.com/stephanenicolas/robospice/issues/406
>
>-- 
>Sent from my Android device with K-9 Mail. Please excuse my brevity.

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by softwaremaverick** [comment]
I'm just pleased to see the modular approach to robospice is seeing a lot more usages.

On 17 February 2015 20:14:33 GMT+00:00, Andrew Clark <andrew@dync.co.uk> wrote:
>Just realised you said protected already so never mind :-)
>
>Just submit a commit with your changes and someone will be right with
>you!
>
>On 17 February 2015 20:12:48 GMT+00:00, Andrew Clark
><andrew@dync.co.uk> wrote:
>>That would be easy enough to do and I doubt anyone would complain.
>>Although you should probably make it protected as opposed to public
>>since you'll only use it in your own spice service.
>>
>>On 17 February 2015 19:50:26 GMT+00:00, Igor Zubenko
>><notifications@github.com> wrote:
>>>Hi, I need to replace DefaultRequestRunner to own. But as I see
>>>SpiceService.createRequestRunner() now is private method.
>>>Is it possible to make it protected?
>>>
>>>---
>>>Reply to this email directly or view it on GitHub:
>>>https://github.com/stephanenicolas/robospice/issues/406
>>
>>-- 
>>Sent from my Android device with K-9 Mail. Please excuse my brevity.
>
>-- 
>Sent from my Android device with K-9 Mail. Please excuse my brevity.

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deadly-cat** [comment]
Thanks for your answer.
I will try to create pull request. But I never do it before. :)

**Comment by softwaremaverick** [comment]
My first time was a learning curve too.

Check out GitHub starter guide for details.

It's well worth it!

On 17 February 2015 20:30:42 GMT+00:00, Igor Zubenko <notifications@github.com> wrote:
>Thanks for your answer.
>I will try to create pull request. But I never do it before. :)
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/stephanenicolas/robospice/issues/406#issuecomment-74746832

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deadly-cat** [comment]
Yep, I'm already understood how to do it. But I have one more question. In what branch I can commit?

**Comment by softwaremaverick** [comment]
Best way is to create a new branch from master. 

Then submit that for pull request.  That way you can make more branches from master with different pull requests

On 17 February 2015 20:41:32 GMT+00:00, Igor Zubenko <notifications@github.com> wrote:
>Yep, I'm already understood how to do it. But I have one more question.
>In what branch I can commit?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/stephanenicolas/robospice/issues/406#issuecomment-74748877

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deadly-cat** [comment]
Ok. Thanks for your help.

**Comment by softwaremaverick** [comment]
Hi!

Take a look at what I've just read, this is so interesting, read more at <http://ntafrenkycro.thedailypainting.com/lnitz?4aelx>

Andrew Clark
